github-dark.css

```diff
@@ -1738,7 +1738,7 @@
   button.tabnav-tab, .rss-subscribe, a.rss-subscribe, table.capped-list .octicon,
   .site-header-menu .nav-item, .discussion-item-icon, .exploregrid-item,
   a.exploregrid-item, .commit-ref .user, .markdown-body blockquote strong {
-    color: #b5b5b5 !important;
+    color: #e2e2e2 !important;
```

Comment:
This makes it more difficult to distinguish the user from the commit message in the latest commit box in repos now. Are we sure we want to make it *that* much brighter?